packages/create-cli/src/utils/package.ts

```diff
@@ -8,7 +9,7 @@ export interface PackageJson {
 }
 
 export function hasPackageJsonFile () {
-  return fs.existsSync('./package.json')
+  return fs.existsSync(path.join(process.cwd(), './package.json'))
```

Comment:
Is the `./` still needed before the file name now that we're also including the path?

Reply:
@shiini2 is not, PR updated